addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -403,7 +404,12 @@ export class HtmlField extends Component {
                 if (urgent && status(this) !== 'destroyed') {
                     await this.updateValue();
                 }
-                await savePendingImagesPromise;
+                try {
+                    await savePendingImagesPromise;
+                } catch (error) {
+                    await this.userService.hasGroup('base.group_portal') ? null : (() => { throw error; })();
+                    // Portal users don't have rights to create attachments. Silently keep base64.
```

Comment:
```suggestion
// Portal users don't have rights to create attachments. Silently keep base64 in that case.
const isPortalUser = await this.userService.hasGroup('base.group_portal');
if (!isPortalUser) {
    throw error;
}
```
Also, it would be better to check that the error is indeed an access rights error, otherwise portal users won't have any error at all, even when we would need to see the error to fix the issue.